src/XMakeCommandLine/UnitTests/CommandLineSwitches_Tests.cs

```diff
@@ -842,7 +842,7 @@ public void SetParameterizedSwitchTests3()
             // parameters, but for testing purposes this is fine
             Assert.True(switches.SetParameterizedSwitch(CommandLineSwitches.ParameterizedSwitch.Logger, "/LOGGER:\"\",asm;\"p,a;r\"", "\"\",asm;\"p,a;r\"", true, false));
 
-            Assert.Equal("/LOGGER:\"\",asm;\"p,a;r\"", switches.GetParameterizedSwitchCommandLineArg(CommandLineSwitches.ParameterizedSwitch.Logger));
+            Assert.Equal("/l:\" /LOGGER:\"\",asm;\"p,a;r\"", switches.GetParameterizedSwitchCommandLineArg(CommandLineSwitches.ParameterizedSwitch.Logger));
```

Comment:
Isn't this an added unclosed quote? That seems odd, since it'd be hard to get on the command line.